### DISCUSSION TITLE
Broken assets path in README after Github flavoured markdown

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#107] [Website] [answered]
Previously. the assets path relative to the repo will be prefixed automatically by `https://raw.githubusercontent.com/<repo-path>/HEAD/` so that we won't need to publish assets to npm.

After Github flavoured markdown change, the assets paths are not prefixed and this lead to broken images in the READMEs. Is the change intentional?

**Comment by radiantspace** [comment, marked as answer]
Hey, can you share the package link (e.g. npmjs.com/package/...) here or via our support form - https://www.npmjs.com/support (while being logged in)?

**Reply by ghost** [reply]
Works here: https://www.npmjs.com/package/merge-json-webpack-plugin/v/5.0.1
Broken: https://www.npmjs.com/package/merge-json-webpack-plugin/v/5.0.2 (latest published yesterday)

**Reply by ethomson** [reply]
Thanks for the details @sibiraj-sr - we're investigating and should have an update 🔜

**Reply by radiantspace** [reply]
@sibiraj-sr we deployed a fix to keep the original relative links behaviour. Your readme should be now properly rendered. Thanks again for sharing those findings.

**Reply by ghost** [reply]
Images are now rendered properly. Thanks 👍
